store/buyProducts_test.js

```diff
@@ -39,10 +39,17 @@ Data(products).Scenario('buy products', async ({I, productPage, current, tryToHe
     let productTotalPrice = await I.getNumericPrice(await productPage.getProductTotalPrice());
     console.log(productTotalPrice);
     I.assertEqual(productTotalPrice, (productPrice + productShipping), 'Prices do not match');
+    
+    let response = await I.sendGetRequest('https://bank.gov.ua/NBUStatService/v1/statdirectory/exchange?valcode=USD&json');
+    let dataRate = response.data[0].rate;
+    console.log(dataRate);
+    let totalPriceUah = productTotalPrice * dataRate;
```

Comment:
create method in helper `convertUSDtoUAH()` which will do GET request and price conversion under the hood